packages/pluggableWidgets/datagrid-dropdown-filter-web/src/DatagridDropdownFilter.editorConfig.ts

```diff
@@ -26,16 +26,16 @@ export const getPreview = (values: DatagridDropdownFilterPreviewProps): Structur
                         children: [
                             {
                                 type: "Text",
-                                fontColor: "#BBBBBB",
+                                fontColor: values.emptyOptionCaption ? "#BBBBBB" : "#FFF",
                                 italic: true,
-                                content: values.emptyOptionCaption ?? ""
+                                content: values.emptyOptionCaption.length > 0 ? values.emptyOptionCaption : "Sample"
```

Comment:
Similar comment here:
```suggestion
                                content: values.emptyOptionCaption ? values.emptyOptionCaption : "Sample"
```

Reply:
Same as my previous comment. I am changing the color, sample here is just to keep the height

Reply:
Fixed

Reply:
@diego-antonelli missed this one 😄

Reply:
Pushed 😅